src/Fantomas.Extras/IgnoreFile.fs

```diff
@@ -31,17 +43,30 @@ module IgnoreFile =
         else
             path
 
-    let isIgnoredFile (file: string) =
-        let fullPath = Path.GetFullPath(file)
+    /// When executed from the command line, Fantomas only supports a `.fantomasignore` file in the
+    /// current working directory. This is that `.fantomasignore` file.
+    let current: Lazy<IgnoreFile option> =
+        lazy
+            let path =
```

Comment:
Perhaps we traverse upwards here as well?
That would cover the scenario where you are inside a subfolder and `.fantomasignore` at the root of the repository is still found.

Reply:
I think you will actually need to do this in order to fix the failing test.